README.md

```diff
@@ -3,7 +3,7 @@
 
 _Cloud Container Attack Tool (CCAT) is a tool for testing security of container environments._
```

Comment:
Can you update the old code in setup.py and wherever else there is old code? "Container Hack Tool" and the wrong GitHub URL and things like that

Reply:
Oh, good catch and I can definitely fix that.

Reply:
I thought I fix that already.